src/Descriptors/Schema/Schema.php

```diff
@@ -402,12 +404,24 @@ protected function attributes(
 
               $schema = $fieldDataType->title($field->name());
 
-              $column = $field instanceof Attribute ? $field->column() : $field->name();
-              if (isset($example[$column])) {
-                  $schema = $schema->example($example[$column]);
-              }
-              if ($field->isReadOnly(null)) {
-                  $schema = $schema->readOnly(true);
+              try {
+                  $column = $field instanceof Eloquent\Fields\Attribute ? $field->column() : $field->name();
+
+                  if ($field instanceof Attribute) {
```

Comment:
It's a little confusing which attribute class this is.

Reply:
I agree, so changed it to use aliases for both attributes